app/models/way.rb

```diff
@@ -2,8 +2,4 @@ class Way < ActiveRecord::Base
   attr_accessible :body, :lat, :lon, :map_id, :color
   has_many :nodes, :dependent => :destroy
 
-  def bbox=(bbox)
-    # counting from left, counter-clockwise
-    self.lon1,self.lat2,self.lon2,self.lat1 = bbox
-  end
 end
```

Comment:
This method is redundant too.
What is the Way model supposed to do anyways?
It is not being used anywhere

Reply:
I think it may be used in annotations, but am not sure! Can you look at the annotations interface which may be a bit broken?

Reply:
But we could address this in another issue later. You could also look at the commits around that time using the history or blame view - see? Yes, this is VERY old! And, i think, can go!

https://github.com/publiclab/mapknitter/blame/e0c305b89a53828006f20de422da2bbffb53a21b/app/models/way.rb